### PR TITLE
[PLG-321] Aqua legacy collector cleanup

### PR DESCRIPTION
**Pull request by mkinchin-paladin**
# Description

MUST be pushed after #1706 
Tons of clean up and style modifications.
Size reduced by 20 times form 290 to 14 mb.

Fixes # (issue)

- [X] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Not tested yet!

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My commit message/PR follows the contribution guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# **Other information**:

List any documentation updates that are needed for the Wiki

### CODE REVIEW

File: jobs/pacman-aqua-enricher/src/main/java/com/tmobile/cso/pacman/aqua/jobs/AquaImageVulnerabilityDataImporter.java [hunk 1]
```diff
@@ -9,125 +23,110 @@
...
-        for (Map<String, Object> res : result) {
-          String idValue = buildIDValue(res);
-          res.putIfAbsent("idKey", idValue);
-          vulnDetails.add(res);
+    private void uploadToES(List<Map<String, Object>> result) throws AquaDataImportException {
+        ObjectMapper objectMapper = new ObjectMapper();
+        objectMapper.setDateFormat(new SimpleDateFormat("yyyy-MM-dd"));
+        List<Map<String, Object>> vulnDetails = new ArrayList<>();
+        try {
+            synchronized (vulnDetails) {
```
Comment (plyubich) [inline jobs/pacman-aqua-enricher/src/main/java/com/tmobile/cso/pacman/aqua/jobs/AquaImageVulnerabilityDataImporter.java:108]: Synchronized is used twice. Do we need it?

File: jobs/pacman-aqua-enricher/src/main/java/com/tmobile/cso/pacman/aqua/jobs/AquaVMVulnerabilityDataImporter.java [hunk 2]
```diff
@@ -10,92 +24,88 @@
...
 
-  /** The Constant index. */
-  private final static String AQUA_VM_VULNERABILITY = "aqua-vm-vulnerability";
-
-  /** The Constant type. */
-  private final static String TYPE = "vulnerability";
+    @SuppressWarnings("unchecked")
+    public Map<String, Object> execute() {
+        try {
+            String token = getBearerToken();
```
Comment (plyubich) [inline jobs/pacman-aqua-enricher/src/main/java/com/tmobile/cso/pacman/aqua/jobs/AquaVMVulnerabilityDataImporter.java:50]: Can we obtain a token once per lifespan of the app and reuse it?

**Review by plyubich — APPROVED** [review]
Minor comments, but looks good as is

**Comment by mkinchin-paladin** [comment]
@plyubich I must say the whole point of this PR is to clean up. What's your comments are about is more refatoring-wise changes. I will get there soon as well, but not in this RP.
Let me create another one for refactoring and address your comments there.

**Pull request merged**